### PR TITLE
CTDA-923/Add details section of audit events

### PR DESCRIPTION
**Pull request by GeorgeColdham**
# CTDA-923/Add details section of audit events

details

[Ticket](https://jira.tools.tax.service.gov.uk/browse/CTDA-923)

### CODE REVIEW

File: test/audit/AuditServiceSpec.scala [hunk 1]
```diff
@@ -51,6 +51,7 @@ class AuditServiceSpec extends AnyFreeSpec with ScalaCheckPropertyChecks with Be
     "must audit notification message event" in {
 
       val requestXml = <xml>test</xml>
+      val requestedXmlToJson = Json.parse("{\"channel\":\"api\",\"xml\":\"test\"}")
 
       val auditDetails = Json.toJson(AuditDetails(Json.obj("xml" -> "test")))
```
Comment (RobertLaverick) [inline test/audit/AuditServiceSpec.scala:56]: Just to check, is this field still used? If not lets clean it up

Comment (GeorgeColdham) [reply test/audit/AuditServiceSpec.scala:56]: It is still used, its the response expected in the test, I could change it to `expectedResponse` if that makes it clearer?

Comment (GeorgeColdham) [reply test/audit/AuditServiceSpec.scala:56]: Was looking at the wrong bit :) Ill remove it!

Comment (GeorgeColdham) [reply test/audit/AuditServiceSpec.scala]: Annnddd removed! :)

**Review by RobertLaverick — APPROVED** [review]
but obviously don't merge till we're allowed to

**Comment by RobertLaverick** [comment]
@GeorgeColdham go ahead and merge this

**Pull request merged**